background.js

```diff
@@ -3,33 +3,33 @@ chrome.browserAction.setBadgeBackgroundColor({color: '#4688F1'});
 let reloads_holder = {};
 
 chrome.storage.onChanged.addListener((whatchanged, area) => {
-    if(area==="local"){
-        let keys = Object.keys(whatchanged);
-        keys.forEach(element => {
-            console.log(whatchanged[element]);
-            if(whatchanged[element].newValue){
-                reloads_holder[element] = setInterval(()=>{
-                    chrome.tabs.reload(parseInt(element));
-                }, 10000);
-            }
-            else {
-                clearInterval(reloads_holder[element]);
+    const currentKey = Object.keys(whatchanged)[0];
+    const currentVal = whatchanged[currentKey].newValue;
+    if(area === "local" && typeof currentVal === 'boolean' ) {
+        chrome.storage.local.get('input', (result) => {
```

Comment:
On the line with another comment in `popup.js`, this logic should change to have a separate stored value for every tab whenever user sets a custom reload interval.

Reply:
> Done. I changed the value stored in `chrome.storage.local` as an object in which the key equals `tab.id` of the tab in question. Its properties are `{ active: bool, input: num }`. That way, each tab has its own set up parameters. I tested this on 5 different tabs and it works simultaneously with their own interval parameters.